include/swift/AST/Attr.h

```diff
@@ -1501,6 +1502,9 @@ class CustomAttr final : public DeclAttribute,
   Expr *getSemanticInit() const { return semanticInit; }
   void setSemanticInit(Expr *expr) { semanticInit = expr; }
 
+  Expr *getInitialValueInit() const { return initialValueInit; }
+  void setInitialValueInit(Expr *expr) { initialValueInit = expr; }
```

Comment:
If we're going to keep this new bit of state, I would prefer that it not be on `CustomAttr`. Rather, it should be part of the semantic information for the specific property wrapper (`PropertyWrapperBackingPropertyInfo`).